keep-ui/app/(keep)/providers/provider-logs.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Card, Title, Badge, Text, Button } from "@tremor/react";
+import { useProviderLogs, ProviderLog } from "@/utils/hooks/useProviderLogs";
+import { EmptyStateCard } from "@/components/ui/EmptyStateCard";
+import { ArrowPathIcon } from "@heroicons/react/24/outline";
+
+interface ProviderLogsProps {
+  providerId: string;
+}
+
+const LOG_LEVEL_COLORS = {
+  INFO: "blue",
+  WARNING: "yellow",
+  ERROR: "red",
+  DEBUG: "gray",
+  CRITICAL: "rose",
+} as const;
+
+const ProviderLogs: React.FC<ProviderLogsProps> = ({ providerId }) => {
+  const { logs, isLoading, error, refresh } = useProviderLogs({ providerId });
+
+  if (isLoading) {
+    return <Text>Loading logs...</Text>;
+  }
+
+  if (error) {
+    return (
+      <div className="flex items-center">
+        <EmptyStateCard
+          title="Provider Logs Not Enabled"
+          description="Provider logs need to be enabled on the backend. Please check the documentation for instructions on how to enable provider logs."
+          buttonText="View Documentation"
+          onClick={() => window.open("https://docs.keephq.dev", "_blank")}
+        />
+      </div>
+    );
+  }
```

Comment:
Can we show this only on 404 and error state otherwise?
something like
```suggestion
  if (error instanceof KeepApiError && error.statusCode === 404) {
      return (
        <div className="flex items-center">
          <EmptyStateCard
            title="Provider Logs Not Enabled"
            description="Provider logs need to be enabled on the backend. Please check the documentation for instructions on how to enable provider logs."
            buttonText="View Documentation"
            onClick={() => window.open("https://docs.keephq.dev", "_blank")}
          />
        </div>
      );
  }
  
  if (error) {
      return (
        <div className="flex items-center">
          <EmptyStateCard
            title="Error Loading Provider Logs"
            description="Seems like we encountred some problem while trying to load provider logs. Please contact us if this issue continues."
            buttonText="Slack Us"
          onClick={() => {
            window.open("https://slack.keephq.dev/", "_blank");
          }}
          />
        </div>
      );

  }
```